docs/content/dagster-cloud/managing-deployments/setting-up-alerts.mdx

```diff
@@ -61,17 +61,6 @@ Currently, Slack and email notifications are supported.
 
    - **Alert policy name** - Enter a name for the alert policy. For example, `slack_urgent_failure`
    - **Description** - Enter a description for the alert policy
-
-   For asset-based alerts, fill out these additional options:
-
-   - **Asset group** - Select the asset group to monitor. You will have the option to select from all asset groups in the deployment.
-   - **Asset key** - Select the asset key to monitor. You will have the option to select from all asset keys in the deployment. **Note:** If you select an asset group, you will not be able to select an asset key.
-   - **Events** - Select whether the alert should trigger on asset materialization failure, asset materialization success, asset check error, asset check warn, asset check passed, or asset check failure to execute
-
-   For job-based alerts, fill out these additional options:
-
-   - **Tags** - Add tag(s) for the alert policy. Jobs with these tags will trigger the alert. For example: `level:critical` or `team:sales`
-   - **Events** - Select whether the alert should trigger on job success, failure, or both
```

Comment:
Moves the notification service options to be in the section with other mandatory fields, given that these notification options apply to all alert types